tests/playwright/pages/Dashboard/Grid/Column/index.ts

```diff
@@ -213,6 +248,21 @@ export class ColumnPageObject extends BasePage {
     }
   }
 
+  async duplicateColumn({ title, expectedTitle = `${title}_copy` }: { title: string; expectedTitle?: string }) {
```

Comment:
If `duplicateColumn` is creating an api call make sure to use `waitForResponse` to await on this.

Reply:
We are verifying by waiting for the toast message and then checking the UI for the column as well.

```js
    await this.verifyToast({ message: 'Column duplicated successfully' });
    await this.grid.get().locator(`th[data-title="${expectedTitle}"]`).isVisible();
```